sphinx/Infinitude_of_Primes.rst

```diff
@@ -5,26 +5,28 @@ This article gives a formal statement of Infinitude of Primes.  For Wikipedia's
 description, see
 `Infinitude of Primes <https://en.wikipedia.org/wiki/Prime_number#Infiniteness>`_.
 
-Informally statement
+Informally statement ::
 
   There are infinitely many :ref:`prime` numbers.
 
 
 Preformally ::
```

Comment:
`Preformally ::` (and `Formally statement ::` below) still use the double-colon syntax for reST code blocks. Now that you've inserted the `.. code-block::` directives, you can get rid of the double colons and just treat them as subsection headers.